src/Articles.jsx

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ArticleList } from "./ArticleList";
+import Lottie from "lottie-react";
+import loading from "./assets/loading.json";
+import { PageSelection } from "./PageSelection";
+import { Filters } from "./Filters";
+
+export const Articles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Really nice to see a loading state, but should it be set to false by default? Are we assuming we do actually have the data on initial load?